c_glib/test/test-sparse-union-scalar.rb

```diff
@@ -49,7 +49,7 @@ def test_equal
   end
 
   def test_to_s
-    assert_equal("...", @scalar.to_s)
+    assert_equal("(2: number: int8 = -29)", @scalar.to_s)
```

Comment:
This is a bit cryptic if you don't know a union is expected. What about e.g. `union{number: int8 = -29}`?